### PR TITLE
Feature#9 integration back front

### PR DESCRIPTION
**Pull request by Camilapinz18**
#Description
Front-end connection to backend

#What did you implemented?
-Changes realized to adapt the front-end integration
-Validators changed
-No functional code deleted

image
image

### CODE REVIEW

File: controllers/login.controller.js [hunk 1]
```diff
@@ -2,13 +2,17 @@ const db = require('../db/db')
 const bcryptjs = require('bcryptjs')
 
 const loginUser = async (request, response) => {
+  console.log("login back")
```
Comment (yovanha) [inline controllers/login.controller.js:5]: please remove your console logs

File: controllers/login.controller.js [hunk 2]
```diff
@@ -2,13 +2,17 @@ const db = require('../db/db')
...
   const { password, email } = request.body
   const [userExists] = await db.query('SELECT * FROM User WHERE email=?;', [
     email
   ])
+
+  console.log(userExists,"exsisi")
```
Comment (yovanha) [inline controllers/login.controller.js:11]: same here

File: controllers/signup.controller.js [hunk 3]
```diff
@@ -1,32 +1,43 @@
...
+  const { user_name, partner_name, date,password, confirmPassword, email } = req.body
+  console.log(
+    'estoy en signup back',
+    'passwrod',
+    password,
+    'passwrodConfirm',
+    confirmPassword,
+    'email',
+    email
+  )
```
Comment (yovanha) [inline controllers/signup.controller.js:14]: same here

File: controllers/signup.controller.js [hunk 4]
```diff
@@ -1,32 +1,43 @@
...
   const [alreadyExists] = await db.query('SELECT * FROM User WHERE email=?', [
     email
   ])
+
   if (alreadyExists.length) {
     res.status(409).send({ message: 'User already exists' })
   } else {
     //encriptar clave:
     if (password === confirmPassword) {
+      console.log('E NOPXISTS____', alreadyExists)
```
Comment (yovanha) [inline controllers/signup.controller.js:25]: same here

**Review by yovanha — CHANGES_REQUESTED** [review]

**Review by garzo94 — APPROVED** [review]

**Review by yovanha — APPROVED** [review]

**Review by arthurchina92 — APPROVED** [review]

**Review by IvanGodinez21 — APPROVED** [review]

**Pull request merged**